src/services/contact.js

```diff
@@ -0,0 +1,29 @@
+import React, { useRef } from 'react';
+import emailjs from '@emailjs/browser';
+
+export const ContactUs = () => {
```

Comment:
You are building a React component called ContactUs here, **what we want is the creation of the reusable JS function for sending e-mails**, not a component.